api/v1alpha1/kogitoserverlessworkflow_types.go

```diff
@@ -331,18 +343,15 @@ type Endpoint struct {
 	Protocol string `json:"protocol,omitempty"` // "TCP" or "UDP"; never empty
 }
 
-type StatusCondition string
-
-const (
-	BuildingStatusCondition StatusCondition = "Building"
-	ReadyStatusCondition    StatusCondition = "Ready"
-)
-
-// KogitoServerlessWorkflowStatus defines the observed state of KogitoServerlessWorkflow
-type KogitoServerlessWorkflowStatus struct {
-	Conditions StatusCondition    `json:"conditions,omitempty"`
-	Endpoints  []Endpoint         `json:"endpoints"`
-	Address    duckv1.Addressable `json:"address,omitempty"`
+// KogitoSWFStatus defines the observed state of KogitoServerlessWorkflow
+type KogitoSWFStatus struct {
```

Comment:
Same thing here. We can use an alias in the CRD to ksw, so we would do `oc get ksw` for short.